activestorage/lib/active_storage/engine.rb

```diff
@@ -34,8 +34,8 @@ class Engine < Rails::Engine # :nodoc:
     end
 
     initializer "active_storage.services" do
-      config.after_initialize do |app|
-        if config_choice = app.config.active_storage.service
+      config.to_prepare do
+        if config_choice = Rails.configuration.active_storage.service
           config_file = Pathname.new(Rails.root.join("config/storage.yml"))
```

Comment:
I’m in favor of extracting a reusable method, but I’d like to do it in a separate PR. There are a few other places where we do roughly the same thing but in slightly different ways, and I’d like to consolidate them.